added compile description to README.md

chihirobelmo opened the pull request:
also added hidden-imports tomlkit as even if I pip install tomlkit it never compiled

Briland5 commented:
The intended environment setup is 
pip install -r requirements.txt
either in a venv or your base environment. This should let you build normally without the hidden import. I agree compilation instructions should be added to the README. Please test the requirements build without the hidden imports and let me know if that fixes the issue for you.
Thanks!

chihirobelmo commented:
yeah, it worked! 
I did not know about Python project setup though why did I not check another .txt at directly :p
thx! I reverted .spec file and updated README.md now.
